Linux/Car_Client/chronoscomm.cpp

```diff
@@ -850,7 +854,7 @@ bool ChronosComm::decodeMsg(quint16 type, quint32 len, QByteArray payload, uint8
                 break;
             }
         }
-        //TODO: handle EXAC message
+    mGpioControl->GPIO_Write(4, 1);
```

Comment:
There should be a check so that this EXAC's actionID matches a previously (successfully) configured ACCM actionID.

Reply:
Added a new task for dealing with message